test/unit/v0/test_bucket.py

```diff
@@ -424,8 +424,8 @@ def test_upload_bytes(self):
 
     def test_upload_bytes_progress(self):
         data = b'hello world'
-        progress_listener = StubProgressListener()
-        self.bucket.upload_bytes(data, 'file1', progress_listener=progress_listener)
+        with StubProgressListener() as progress_listener:
```

Comment:
this kind of shows that the interface for self.bucket.upload_bytes has changed, as now the caller is responsible for entering the context manager. I think we will need an adapter for that.

Reply:
Yeah, about that. I don't really want to create v2 for that. For me, it's a bug as the documentation clearly shows that the progress listener should be used as a context manager. But it's a bug that the solution is kind of NBC

I think the best is to skip it for now, and I can create an issue for that. It's should be in the backlog for v2. Do we have sth like that?

Reply:
https://github.com/Backblaze/b2-sdk-python/issues/139

Reply:
Reverted